src/dotnet-scaffolding/Microsoft.DotNet.Scaffolding.Core/Scaffolders/Scaffolder.cs

```diff
@@ -39,9 +39,12 @@ public async Task ExecuteAsync(ScaffolderContext context)
         {
             var step = _steps[stepIndex];
             var preparer = _preparers[stepIndex];
-
             preparer.RunPreExecute(step, context);
-            await step.ExecuteAsync(context);
+            if (!step.SkipStep)
```

Comment:
I'm still not entirely sure about the SkipStep property. But we can iterate.

Reply:
sounds good